src/register-org/components/document-exchange-reference-details/document-exchange-reference-details.component.ts

```diff
@@ -12,7 +12,7 @@ export class DocumentExchangeReferenceDetailsComponent extends RegisterComponent
   public dxFormGroup: FormGroup;
 
   constructor(public readonly router: Router,
-    public readonly registerOrgService: RegisterOrgService,
+    public readonly registerOrgService: RegisterOrgService
```

Comment:
@johnbenjamin-hmcts - does this service need to be `public`? Same for the router injection above

Reply:
@Daniel-Lam - All the components extend from the master page component called `RegisterComponent` that takes in Router and RegisterOrgService as the constructor properties.

Having them as private on both the master page component and child components did not work.